src/psyclone/dynamo0p3.py

```diff
@@ -6047,6 +6463,10 @@ def __init__(self, kernel_args, arg_meta_data, arg_info, call):
         self._vector_size = arg_meta_data.vector_size
         self._type = arg_meta_data.type
         self._stencil = None
+        if arg_meta_data.mesh:
```

Comment:
parent method need docstring interface information.

Reply:
Done. I think this is precisely the kind of information I needed when I first started working on PSyclone so it feels good to add it :-)